src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.Todo;
+import exception.DukeException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    static List<Task> tasks = new ArrayList<>();
+    static Boolean notBye = true;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+//        String logo = " ____        _        \n"
+//                + "|  _ \\ _   _| | _____ \n"
+//                + "| | | | | | | |/ / _ \\\n"
+//                + "| |_| | |_| |   <  __/\n"
+//                + "|____/ \\__,_|_|\\_\\___|\n";
+        String greeting = "Hello I'm Duke\n"
+                + "What can I do for you?";
+        System.out.println(greeting);
+
+        Scanner sc = new Scanner(System.in);
+        String userInput;
+        while(notBye) {
+            userInput = sc.nextLine();
+            try {
+                processUserInput(userInput);
+            }catch (DukeException e){
+                System.out.println(e.toString());
+            }
+            System.out.println("-----------------------------");
+
+        }
+    }
+
+    /**
+     * process the user input and understand the command
+     * @param userInput: value input by a user
+     */
+    private static void processUserInput(String userInput) throws DukeException {
+        String description = null;
+        if(userInput.startsWith("todo")){
+            if(userInput.equals("todo")){
+                throw new DukeException("OOPS!!! The description of a todo cannot be empty.");
+            }
+            else {
+                description = userInput.substring(5);
+                tasks.add(new Todo(description));
+            }
+            }
```

Comment:
Though not affecting the runtime, this curly bracket indentation could be misleading at first glance.  It seems you meant for it to close the block starting from line 45 instead of line 44.  Consider reviewing this portion of code to increase readability.